docs/EARLY_TESTERS.md

```diff
@@ -0,0 +1,14 @@
+# EARLY TESTERS PROGRAMME
+
+## What is it?
+
+## What are the expectations?
+
+## Who has signed up?
+
+- Infura
+- Textile
```

Comment:
Let's add contacts here (i.e., github handles).

Reply:
@MichaelMure @sanderpick - are you folks on board with helping us test out new releases?

Reply:
As of right now, Infura doesn't have a way to do canary testing, but we are aiming for it. Please send us notification and we'll get to proper testing eventually.

Reply:
We don’t have an automated way to try different releases but we can certainly try all the candidates as they come out.

Reply:
Having RTrade/Temporal added to the list would be helpful as well and save us some trouble :)